lldb/tools/lldb-dap/DAP.h

```diff
@@ -152,7 +152,7 @@ struct DAP {
   llvm::DenseSet<ClientFeature> clientFeatures;
 
   /// The initial thread list upon attaching.
-  std::optional<llvm::json::Array> initial_thread_list;
+  std::optional<std::vector<protocol::Thread>> initial_thread_list;
```

Comment:
Is it worth distinguishing between an empty list and not having the list at all?

Reply:
Changed to just a vector that we check.